scripts/release/release-ckeditor5.js

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* eslint-env node */
+
+'use strict';
+
+const path = require( 'path' );
+const { tools, logger } = require( '@ckeditor/ckeditor5-dev-utils' );
+const versionUtils = require( '@ckeditor/ckeditor5-dev-env/lib/release-tools/utils/versions' );
+const cli = require( '@ckeditor/ckeditor5-dev-env/lib/release-tools/utils/cli' );
+const createGithubRelease = require( '@ckeditor/ckeditor5-dev-env/lib/release-tools/utils/creategithubrelease' );
+const validatePackageToRelease = require( '@ckeditor/ckeditor5-dev-env/lib/release-tools/utils/validatepackagetorelease' );
+const { getChangesForVersion } = require( '@ckeditor/ckeditor5-dev-env/lib/release-tools/utils/changelog' );
+
+const log = logger();
+const packageJsonPath = path.resolve( __dirname, '..', '..', 'package.json' );
+let packageJsonCopy;
+
+cli.provideToken()
+	.then( token => {
+		const gitVersion = versionUtils.getLastTagFromGit();
+		const changelogVersion = versionUtils.getLastFromChangelog();
+
+		log.info( 'Comparing versions - saved in changelog and as a tag...' );
+
+		// If the last tag is equal to version saved in changelog, we don't have new version for release.
+		if ( gitVersion === changelogVersion ) {
+			return reject( 'Before starting the release process, you should generate the changelog.' );
+		}
+
+		const releaseDescription = getChangesForVersion( changelogVersion );
+
+		log.info( 'Validating the repository for the release...' );
+
+		const errors = validatePackageToRelease( {
+			version: changelogVersion,
+			changes: releaseDescription
+		} );
+
+		// Abort due to errors during validation.
+		if ( errors.length ) {
+			const log = logger();
+
+			log.error( 'Unexpected errors occured:' );
+			errors.map( err => '* ' + err ).forEach( log.error.bind( log ) );
+
+			return reject( 'Releasing has been aborted due to errors.' );
+		}
+
+		log.info( 'Bumping the version...' );
+
+		// Bump the version.
+		tools.shExec( `npm version ${ changelogVersion } --message "Release: v${ changelogVersion }."`, { verbosity: 'error' } );
+
+		log.info( 'Updating the package.json...' );
+
+		// Update the `package.json`. Published package shouldn't have any dependencies and devDependencies.
+		// Also, we should omit our environment scripts and their configs.
+		tools.updateJSONFile( packageJsonPath, packageJson => {
```

Comment:
I'd rather preferred if we had all the files that are used during a release in a separate directory. I don't like changing existing files because e.g. if we'll add some new property to the main `package.json` we will need to remember to update this script.

So, it's better to have templates of these files in some other directory (`scripts/release/template`) and modify them there (e.g. copying some props from the main `package.json`.

Reply:
I created a template with all required properties. They are set by default as `null`. It means that we will copy values from original (source) `package.json`. If a value is other than `null`, the script won't touch it.

I leave the `"files"` section in the release script because we cannot put a comment in JSON file and I feel that there should be some explanation.